packages/web-components/fast-foundation/src/tree-item/tree-item.ts

```diff
@@ -140,19 +150,19 @@ export class TreeItem extends FASTElement {
         }
     }
 
-    public handleFocus = (e: Event): void => {
-        if (e.target === e.currentTarget) {
-            this.focusable = true;
-        }
-    };
-
-    public handleBlur = (e: FocusEvent): void => {
-        if (e.target !== e.currentTarget) {
-            return;
-        }
+    /**
+     * @deprecated - no longer needed.
+     * @param e - Event object
+     */
+    /* eslint-disable-next-line */
+    public handleFocus = (e: Event): void => {};
 
-        this.focusable = false;
```

Comment:
We are using this kind of focusing process on other components, or we have at least. Should we consider staying away from this sort of pattern in Web components now?

I recall using it on a react grid component, so just curious as to your thoughts on using this type of pattern elsewhere in web component world.

Reply:
I don't know that it is a universal rule, the problem only exists for for custom elements that are nested in other custom elements, where the parent has `[tabindex=-1]` and the child has `[tabindex=0]`. Non-nested scenarios work as normal